doc/unison-manual.tex

```diff
@@ -2001,18 +2001,20 @@
 ``Ctrl-D'' is expected to send an EOF if the sandard input is a terminal.
 Closing the input and receiving an EOF, a \verb|^C| (0x03), or a \verb|^D|
 (0x04) from the input terminal or a redirected standard input have the same
-effect. If the standard input is redirected from a regular file, not open or
-not open for reading already at Unison startup then it is ignored and only
-signals can be used to send the stop request.
+effect. EOF is not interpreted as a stop request when standard input is
+redirected from a regular file. If the standard input is not open or not open
+for reading already at Unison startup then it is ignored and only signals can
+be used to send the stop request.
 
 \begin{quote}
   {\em Tips:} For continuous synchronization the input should not be
   redirected from \verb|/dev/null| (as that would stop the process after
   the first synchronization). This is what some implementations of \verb|nohup|
   may do if run from a terminal without redirecting input. Input should
   neither be redirected from any source providing a lot of input (such as
-  a device or a pipe that produces large quantities of data). In such cases,
-  Unison would keep reading all the input looking for the stop condition.
+  an existing regular file larger than a few tens of bytes or a device or
+  a pipe that produces large quantities of data). In such cases, Unison
+  would keep reading all the input looking for the stop condition.
```

Comment:
I am not following.   What is the stop condition then.

This seems like a one-off ad hoc mechanism.   It feels like the beginning of a control socket to be able to do things to a long-running background process that has no UI.  There's a long history of that in various programs, and (unix-centric view I know) and the usual solution is some simple protocol on a unix-domain socket.   That leads to wanting to write the pid, and figure out naming because one could run lots of copies.

I lean to adding a mechanism and using it vs a non-extendable mechanism.  But really if you can send SIGINT twice, or SIGUSR2 once, that might be enough.